infrastructure/keyvault.tf

```diff
@@ -0,0 +1,10 @@
+module "azurekeyvault" {
+  source                  = "git@github.com:hmcts/cnp-module-key-vault?ref=master"
+  name                    = "sds${var.product}-${var.environment}"
+  product                 = "${var.product}"
+  env                     = "${var.environment}"
+  tenant_id               = data.azurerm_client_config.current.tenant_id
+  object_id               = azurerm_user_assigned_identity.managed_identity.principal_id
```

Comment:
this is supposed to be the creator,

probably best to change it so that you pass:
`create_managed_identity = true` 

that way the module will create the identity and add permissions to it